Modules/CaPTkSurvival/cmdapps/CaPTkSurvivalPredictionCmdApp.cpp

```diff
@@ -45,19 +45,19 @@ int main(int argc, char* argv[])
             See mitkCommandLineParser::addArgument() for more information. ****/
 
     parser.addArgument(
-                "mode",
-                "mo",
-                mitkCommandLineParser::String,
-                "Mode",
-                "Select which operation to perform. Modes: train or predict",
+                "type",
+                "t",
+                mitkCommandLineParser::Bool,
```

Comment:
Since type is boolean, it should be called "train". In general, type doesn't make sense as a bool

- Test is what the user would usually do. if he provides this argument, that means he wants to test.

Reply:
Note that if you go the "test" bool arg route, then the if/else block in my other comment should be reversed